help/license.md

```diff
@@ -91,6 +84,14 @@ information of many publishers, see the
 Copyright notices
 -----------------
 
+It is usually the case that PDFs found on publisher websites or supplied
+as proofs are the property of the publisher, which often owns the
+copyright and/or licenses their use. Submission of PDFs with copyright
+statements which prohibit or impair arXiv's [redistribution
+license](/help/license) will be rejected. If explicit permission is
+given for open access redistribution, these are permissible.
+Users should contact arXiv administrators at help@arxiv.org if they have questions.

```

Comment:
I thought we were trying to explicitly remove using email addresses in the text of these, in favor of linking to contact?

Reply:
I want to be more specific here for now to direct users away from the copyright infringement contact info